ducktape/tests/test.py

```diff
@@ -133,6 +134,9 @@ def _escape_pathname(s):
     return re.sub("^\.|\.$", "", s)
 
 
+CLUSTER_SIZE_KEYWORD = "num_nodes"
```

Comment:
nit, but should this be here or defined alongside the decorator? Possibly related to question about positional vs keyword args above.